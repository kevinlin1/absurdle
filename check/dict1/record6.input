heats
mends
river
baker
easel
lurch
foggy
folly
jolly
wooly
