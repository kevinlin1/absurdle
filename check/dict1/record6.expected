2309 words
> heats
: ⬜⬜⬜⬜⬜

282 words
> mends
: ⬜⬜⬜⬜⬜

103 words
> river
: ⬜⬜⬜⬜⬜

34 words
> baker
: ⬜⬜⬜⬜⬜

20 words
> easel
: ⬜⬜⬜⬜🟨

12 words
> lurch
: 🟨⬜⬜⬜⬜

5 words
> foggy
: ⬜🟩⬜⬜🟩

2 words
> folly
: ⬜🟩⬜🟩🟩

1 words
> jolly
: ⬜🟩⬜🟩🟩

1 words
> wooly
: 🟩🟩🟩🟩🟩

Absurdle 10/∞

⬜⬜⬜⬜⬜
⬜⬜⬜⬜⬜
⬜⬜⬜⬜⬜
⬜⬜⬜⬜⬜
⬜⬜⬜⬜🟨
🟨⬜⬜⬜⬜
⬜🟩⬜⬜🟩
⬜🟩⬜🟩🟩
⬜🟩⬜🟩🟩
🟩🟩🟩🟩🟩
